### DISCUSSION TITLE
Remove Bed Level Calibration from PRINT_START?

### DISCUSSION THREAD

**Discussion by JoeBobtx** [jschuh/klipper-macros#166] [Q&A]
Please forgive my ignorance as I am new to Klipper. I downloaded and installed the macros and configured my slicer (Orca), however every time I start to try to print it goes through a bed level and then cancels the print. I have already leveled the bed through MainSail, what/where do I need to remove to stop it from releveling the bed at the beginning of each print? Thank You so much.

**Comment by jschuh** [comment]
All the PRINT_START options are [described here](https://github.com/jschuh/klipper-macros#print-start-and-end). You want to look at `variable_start_level_bed_at_temp`.

**Reply by JoeBobtx** [reply]
Thank You. I think I have that part down, but it unfortunately does not preheat the extruder and bed even though I believe I've copied and pasted according to your instructions. Here's what my Start G-Code is...


```
M190 S0 ; Remove this if autoemit_temperature_commands is off in Prusa Slicer 2.6 and later
M109 S0 ; Remove this if autoemit_temperature_commands is off in Prusa Slicer 2.6 and later
G28 ; Home the printer first
_PRINT_START_PHASE_INIT EXTRUDER={first_layer_temperature[initial_tool]}  BED=[first_layer_bed_temperature] LAYERS={total_layer_count} NOZZLE_SIZE={nozzle_diameter[0]}
; Insert custom gcode here.
__PRINT_START_PHASE_PREHEAT
; Insert custom gcode here.
_PRINT_START_PHASE_EXTRUDER
; Insert custom gcode here.
_PRINT_START_PHASE_PURGE
BED_MESH_PROFILE LOAD=08-17

; This is the place to put slicer purge lines if you haven't set a non-zero
; variable_start_purge_length to have START_PRINT automatically calculate and 
; perform the purge (e.g. if using a Mosaic Palette, which requires the slicer
; to generate the purge).`
```

It's also zeroing out the extruder and heated temps about 30 seconds into a print??

**Comment by jschuh** [comment]
Then your slicer is emitting something additional in the startup gcode. Maybe when you pasted in the gcode from the instructions you left some old gcode after it; or maybe there's a slicer setting you need to change to prevent it from inserting the problematic gcode.

If you open up the gcode file in a text editor you'll be able to see the exact gcode that is emitted, which should help you in tracking down your misconfiguration issue.

**Comment by JoeBobtx** [comment]
Thank you for the assistance. I’m green on Klipper and just trying to figure this all out. While there is a lot of information on this stuff, the one thing that’s missing is examples of syntax. I will check it out and did not intend to seek full technical support. LOL. I may just start over with a fresh install since I’ve lost track of the changes made.On Aug 18, 2023, at 8:00 AM, Justin Schuh ***@***.***> wrote:﻿
Then your slicer is emitting something additional in the startup gcode. Maybe when you pasted in the gcode from the instructions you left some old gcode after it; or maybe there's a slicer setting you need to change to prevent it from inserting the problematic gcode.
If you open up the gcode file in a text editor you'll be able to see the exact gcode that is emitted, which should help you in tracking down your misconfiguration issue.
